hack/jenkins/common.sh

```diff
@@ -354,17 +355,6 @@ echo ">> Copying ${TEST_OUT} to gs://${JOB_GCS_BUCKET}out.txt"
 gsutil -qm cp "${TEST_OUT}" "gs://${JOB_GCS_BUCKET}out.txt"
 
 
-echo ">> Attmpting to convert test logs to json"
-if test -f "${JSON_OUT}"; then
-  rm "${JSON_OUT}" || true # clean up previous runs of same build
-fi
-
-touch "${JSON_OUT}"
```

Comment:
How about move this touch to above. I dont know why I had to do this, But I remember it was needed for some odd situations...

Reply:
Now that it is using a file redirect, the file is _always_ created. But note also that "" is not legal JSON (but file is a json stream)

Reply:
Actually it was always doing a redirect before, so the file should always have been created.

```
> "${JSON_OUT}" 
```

Reply:
Can leave it in, but I don't think it has any effect.